projects/storefrontlib/cms-structure/routing/default-routing-config.ts

```diff
@@ -8,18 +8,20 @@ import { FeatureToggles, RoutingConfig } from '@spartacus/core';
 import { inject } from '@angular/core';
 
 export const defaultRoutesConfigFactory: () => RoutingConfig = () => {
+  const featureToggles = inject(FeatureToggles);
   const enableProductPageRouteAllowsNoProductName =
-    inject(FeatureToggles).defaultProductPageRouteAllowsNoProductName;
+    featureToggles.defaultProductPageRouteAllowsNoProductName;
+  const isNewAuthFlow = featureToggles.authorizationCodeFlowByDefault;
```

Comment:
`medium:` The `isNewAuthFlow` does not fully reflect what the `authorizationCodeFlowByDefault` flag is responsible for. In addition, the name is not quite true, as Code Flow has been available in Spartacus for a long time. Let's consider renaming it or simply using `featureToggles.authorizationCodeFlowByDefault` directly.

```suggestion
  const authorizationCodeFlowByDefault = featureToggles.authorizationCodeFlowByDefault;
```